Makefile

```diff
@@ -140,14 +140,14 @@ DPI_CXXFLAGS ?= $(CXXFLAGS) $(DPI_FLAGS) -D_GLIBCXX_USE_CXX11_ABI=0
 
 # this list contains the standalone components
 src :=  core/include/$(target)_config_pkg.sv                                         \
-        corev_apu/tb/ariane.sv                                                       \
-        $(wildcard corev_apu/bootrom/*.sv)                                           \
-        $(wildcard corev_apu/clint/*.sv)                                             \
-        $(wildcard corev_apu/fpga/src/axi2apb/src/*.sv)                              \
-        $(wildcard corev_apu/fpga/src/apb_timer/*.sv)                                \
-        $(wildcard corev_apu/fpga/src/axi_slice/src/*.sv)                            \
-        $(wildcard corev_apu/src/axi_riscv_atomics/src/*.sv)                         \
-        $(wildcard corev_apu/axi_mem_if/src/*.sv)                                    \
+        corev_apu/tb/ariane.sv                                                      \
+        $(subst $(root-dir),,$(wildcard $(root-dir)corev_apu/bootrom/*.sv))                   \
+        $(subst $(root-dir),,$(wildcard $(root-dir)corev_apu/clint/*.sv))                     \
+        $(subst $(root-dir),,$(wildcard $(root-dir)corev_apu/fpga/src/axi2apb/src/*.sv))      \
+        $(subst $(root-dir),,$(wildcard $(root-dir)corev_apu/fpga/src/apb_timer/*.sv))        \
+        $(subst $(root-dir),,$(wildcard $(root-dir)corev_apu/fpga/src/axi_slice/src/*.sv))    \
+        $(subst $(root-dir),,$(wildcard $(root-dir)corev_apu/src/axi_riscv_atomics/src/*.sv)) \
+        $(subst $(root-dir),,$(wildcard $(root-dir)corev_apu/axi_mem_if/src/*.sv))            \
```

Comment:
We include this Makefile from another location when compiling the source files for culsans. Then `$(wildcard corev_apu/*)` returns nothing since `corev_apu` is not present in the current directory. So this change adds `$(root-dir)` before the `$(wildcard`, which then gives a file list as result, then removes the `$(root-dir)` again using `$(subst`.

We could revert this and add the wildcard files explicitly in "our" Makefile, but it feels a bit ugly.

Reply:
I think we can keep this for now. In the long run, we will stop maintaining the Makefile on the pulp branch and switch to [Cheshire](https://github.com/pulp-platform/cheshire) for system-level simulation/emulation (but we can handle this later)